TACC/Frontera-Portal
# bug/FP-830: Fix app cloning

## Overview: ##
When running a cloned app (like zippy), ensure the app and system are functional. If not, recreate the app, or recreate the system as needed.

## PR Status: ##


## Related Jira tickets: ##

* [FP-830](https://jira.tacc.utexas.edu/browse/FP-830)


## Testing Steps: ##
1. Run a known broken cloned app, or break your app's system by disabling it
2. Run the app
3. Ensure it recreates the system/app and runs successfully
